internal/wazeroir/compiler.go

```diff
@@ -3015,13 +3018,14 @@ func (c *compiler) stackPush(ts UnsignedType) {
 func (c *compiler) emit(ops ...Operation) {
```

Comment:
I guess this can accept UnionOperation now
```suggestion
func (c *compiler) emit(ops ...UnionOperation) {
```